setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
 exclude = /**/migrations
-ignore = F401,F821,F841,E402,W503
+ignore = F401,F841,E402,W503
```

Comment:
F401 - modules are imported but not used.
F841 - local variable is defined/assigned but not used
E402 - module import is not at the top of the file
W503 - line break before binary operator

F401 is there as sometimes modules are imported for side effects.  This seems to the be case in atmosphere as there are several files that only contain imports but no code after the import.

F841 is there as there are many such cases (99 cases).  Cannot generally eliminate the line as many function calls have side effects.  Since they are set and not used there is no harm with having them there, just clutters the symbol table.

E402 is still there as both django and atmosphere have modules that imported half way down the file.  django does this to make the file processing easier for their automated tools.

W503 is a religious issue.  Currently it is warning about code that is the preferred PEP8 way, though PEP8 allows for a line break before or after a binary operator.

I am ignoring line length for now.